From 147270f08ac33d737405afc555b3ddb6f1308336 Mon Sep 17 00:00:00 2001
From: Pablo Mendez Hernandez <pablomh@gmail.com>
Date: Wed, 8 Aug 2018 21:02:55 +0200
Subject: [PATCH] gps: use TARGET_BOARD_AUTO to override qcom hals

BOARD_IS_AUTOMOTIVE := true includes car packages on not car devices
see https://android.googlesource.com/platform/packages/services/Car/+/master/car-support-lib/Android.mk#50

Test: car packages can be included.

Change-Id: I28898df1e8855347129039b5cb0d43975d3a5415
Signed-off-by: David Viteri <davidteri91@gmail.com>
---
 Android.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index 2c74106..f85651e 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,5 +1,5 @@
 # TODO:  Find a better way to separate build configs for ADP vs non-ADP devices
-ifneq ($(BOARD_IS_AUTOMOTIVE),true)
+ifneq ($(TARGET_BOARD_AUTO),true)
   ifneq ($(BOARD_VENDOR_QCOM_GPS_LOC_API_HARDWARE),)
     LOCAL_PATH := $(call my-dir)
     ifeq ($(BOARD_VENDOR_QCOM_LOC_PDK_FEATURE_SET),true)
@@ -38,4 +38,4 @@ ifneq ($(BOARD_IS_AUTOMOTIVE),true)
     endif #BOARD_VENDOR_QCOM_LOC_PDK_FEATURE_SET
 
   endif #BOARD_VENDOR_QCOM_GPS_LOC_API_HARDWARE
-endif
+endif #TARGET_BOARD_AUTO
-- 
2.13.0

